content/python/concepts/math-module/terms/math-radians/math-radians.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: 'math.radians()'
+Description: 'Returns the radian equivalent of an input angle given in degress.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Geometry'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **math.radians()** function takes in an angle measured in degrees and returns its radian equivalent.
+
+## Syntax
+
+```shell
+math.radians(x)
+```
+
+`math.radians()` takes in a numeric value or expression `x`, which represents an angle measured in degrees, and returns a floating point value representing the radian equivalent to `x`.
+
+!["45 degree rotations expressed in radian measure" by Adrignola is marked with CC0 1.0. To view the terms, visit https://creativecommons.org/publicdomain/zero/1.0/deed.en?ref=openverse.](https://raw.githubusercontent.com/Codecademy/docs/main/media/radian-measure-45-degree-rotations-256.png)
```

Comment:
Can you please move this image file on this branch of the PR?

Reply:
resolved in f5798cc

Reply:
resolved in 34010f4